src/software/ai/hl/stp/play/penalty_kick_enemy_play.cpp

```diff
@@ -23,33 +25,48 @@ bool PenaltyKickEnemyPlay::invariantHolds(const World &world) const
 void PenaltyKickEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield,
                                           const World &world)
 {
+    auto move_to_goal_line_tactic = std::make_shared<MoveToGoalLineTactic>();
+    std::shared_ptr<const GoalieTacticConfig> goalie_tactic_config =
+        std::make_shared<const GoalieTacticConfig>();
+    auto goalie_tactic = std::make_shared<GoalieTactic>(goalie_tactic_config);
     auto move_tactic_2 = std::make_shared<MoveTactic>(true);
     auto move_tactic_3 = std::make_shared<MoveTactic>(true);
     auto move_tactic_4 = std::make_shared<MoveTactic>(true);
     auto move_tactic_5 = std::make_shared<MoveTactic>(true);
     auto move_tactic_6 = std::make_shared<MoveTactic>(true);
 
+    PriorityTacticVector tactics_to_run = {{move_to_goal_line_tactic, move_tactic_2,
+                                            move_tactic_3, move_tactic_4, move_tactic_5,
+                                            move_tactic_6}};
+
     do
     {
         // Move all non-shooter robots to the center of the field
         move_tactic_2->updateControlParams(
-            Point(0, 0), world.field().enemyGoalCenter().toVector().orientation(), 0);
+            Point(world.field().enemyPenaltyMark().x() + 1.5, 0),
```

Comment:
Is there a reason it has to be 1.5 past the enemy penalty mark? Or is this just an arbitrary number?

Reply:
The rule is that the robots have to be a meter behind the ball. I made it 1.5 for some buffer in case the enemy team decides to place their robots exactly 1 meter behind the ball - in which case we would collide, and the move tactic might move our robots closer to the ball to avoid collisions.

Reply:
Hmm, what if the enemy team doesn't place their robots 1 meter behind the ball but 1.5 meters? I'm not sure how big this problem would be or if it even is a problem? @jonathanlew do you have any comments on this?

Reply:
I think in the worst case when the enemy robots are also exactly 1.5m behind the ball AND we choose to move in front, then it still should be ok since max robot radius is only 0.09m - we'll have a 0.50m buffer to fit 0.27m of robots. If the enemy robots were any closer than 1.5m then our move tactic would move us at least 1.5m behind the ball. Let's see what Jon thinks though because I could be wrong with regards to move tactic behaviour.

Reply:
we can add a simulated test with enemy robots at 1m, 1.4m, 1.5m, 1.6m, etc. behind the ball and check to make sure that our robots do stay 1m behind the ball